LinqToRest/StringToEnumExtension.cs

```diff
@@ -0,0 +1,44 @@
+﻿using System;
+using System.ComponentModel;
+using System.Reflection;
+using Funcky.Extensions;
+
+namespace Messerli.LinqToRest
+{
+    public static class StringToEnumExtension
+    {
+        public static object TryParseToEnumElement(this string candidate, Type type)

```

Comment:
IIRC, the standard library uses the Try prefix to indicate that a function returns a bool and has an out parameter. If that's true, then this method has an unfitting name because it breaks this convention. I could be wrong however.

Reply:
Update: I was right. It's called the TryGet pattern.

Reply:
Renamed to `ParseToEnumElement` in https://github.com/messerli-informatik-ag/linq-to-rest/pull/15/commits/1645eb04a346cbac73a5fb3f33c7f8720d6df12d

Reply:
The variable is still named `tryParse`. Also, what do you think about `ParseAsEnumElement`?

Reply:
addressed inn https://github.com/messerli-informatik-ag/linq-to-rest/pull/15/commits/171f45bb12aad47ca4492a535af7517eeb5d4dd7